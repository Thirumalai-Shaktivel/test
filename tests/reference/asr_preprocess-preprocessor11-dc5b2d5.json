{
    "basename": "asr_preprocess-preprocessor11-dc5b2d5",
    "cmd": "lfortran --cpp --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/preprocessor11.f90",
    "infile_hash": "59524bf0e93e3be3ef12b0b077a8735fc81b4968e674584c018bd18c",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr_preprocess-preprocessor11-dc5b2d5.stdout",
    "stdout_hash": "d227693281ba202c0f1a21ae3ae32c082e8aa151a9dcbeb1cd7f918c",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}