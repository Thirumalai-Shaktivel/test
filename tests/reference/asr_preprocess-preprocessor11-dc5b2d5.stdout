(TranslationUnit (SymbolTable 1 {preprocessor11: (Program (SymbolTable 2 {}) preprocessor11 [] [(Print () [(BinOp (ConstantInteger 1 (Integer 4 [])) Add (BinOp (ConstantInteger 5 (Integer 4 [])) Mul (ConstantInteger 3 (Integer 4 [])) (Integer 4 []) (ConstantInteger 15 (Integer 4 [])) ()) (Integer 4 []) (ConstantInteger 16 (Integer 4 [])) ())]) (Print () [(ConstantString "X is 1" (Character 1 6 () []))]) (Print () [(ConstantString "X is not 2" (Character 1 10 () []))]) (Print () [(ConstantString "Not X is not 1" (Character 1 14 () []))]) (Print () [(ConstantString "X is 1 and Y is 5" (Character 1 17 () []))]) (Print () [(ConstantString "Not (X is 2 and Y is 5)" (Character 1 23 () []))]) (Print () [(ConstantString "X<3 and Y>3" (Character 1 11 () []))]) (Print () [(ConstantString "Not (X>3 and Y<3)" (Character 1 17 () []))]) (Print () [(ConstantString "X is 3-2" (Character 1 8 () []))]) (Print () [(ConstantString "X is not 3-1" (Character 1 12 () []))]) (Print () [(ConstantString "Y is (6*5-5)/5" (Character 1 14 () []))]) (Print () [(ConstantString "Y is not (6*5-6)/5" (Character 1 18 () []))]) (Print () [(ConstantString "Y is (-5+6*5)/5" (Character 1 15 () []))]) (Print () [(ConstantString "Y is not (-6+6*5)/5" (Character 1 19 () []))]) (Print () [(ConstantString "Y is (+5+6*5)/7" (Character 1 15 () []))]) (Print () [(ConstantString "Y is not (+4+6*5)/7" (Character 1 19 () []))]) (Print () [(ConstantString "Y is (+5+6*Y)/7" (Character 1 15 () []))]) (Print () [(ConstantString "Z is 16" (Character 1 7 () []))])])}) [])
